uber/models.py

```diff
@@ -1338,6 +1345,16 @@ def requested(self, department):
     def assigned_to(self, department):
         return int(department or 0) in self.assigned_depts_ints
 
+    def trusted_in(self, department):
+        return int(department or 0) in self.trusted_depts_ints
+
+    def assigned_and_trusted_in(self, department):
```

Comment:
I'm really not sure this is logically necessary - in what possible situation would someone be trusted in a department but not assigned to it?